strawberry/extensions/tracing/datadog.py

```diff
@@ -36,6 +37,31 @@ def _resource_name(self):
 
         return query_hash
 
+    def _create_span(
```

Comment:
Why is that `_`private if it is intended to be overridden? Am I missing something?

Reply:
Good call, I think this should be public.